lca_website/ngo/models.py

```diff
@@ -1,3 +1,27 @@
 from django.db import models
 
-# Create your models here.
+TYPE_ZERO_NGO = 0
+TYPE_ONE_NGO = 1
+
+NGO_CATEGORY = (
```

Comment:
Rename as `NGO_CATEGORIES` since plural.

Read about Enum classes for django and see if using them makes sense to you:
1. https://hackernoon.com/using-enum-as-model-field-choice-in-django-92d8b97aaa63
2. https://medium.com/@bencleary/using-enums-as-django-model-choices-96c4cbb78b2e